dietpi/dietpi-software

```diff
@@ -7552,10 +7642,10 @@ case "$1" in
 		fi
 	;;
 
-        *)
-                echo "[FAILED] Invalid command ($1), please use \"start\" or \"stop\""
-                exit 1
-        ;;
+	*)
+		echo "[FAILED] Invalid command ($1), please use \"start\" or \"stop\""
+		exit 1
+	;;
```

Comment:
That is questionable, can't understand what exactly was changed?

Reply:
Only replaced 8 spaces with 1 tab, to align with other lines in that script.